routes/views/director_profile.js

```diff
@@ -16,9 +16,33 @@ exports = module.exports = (req, res) => {
 
       if (err) return next(err);
 
-      locals.director = director;
+      // find out which fields in this model are translatable fields
+      // don't want to hardcode this, as I would like to export this function and use it across all routes
+      // also translatable key names have pattern name___LOCALE << yes, 3 underscores!!
+      const translatableFields = Object.keys(director._doc)
```

Comment:
I assume ._doc is a copy of all of the data? 
Will all query results have a ._doc copy (for other model queries)?